### ISSUE TITLE
Round to nearest X of interval

### ISSUE THREAD

**Issue by LandonSchropp** [date-fns/date-fns#787]
Sorry if this has already been answered, but I searched the issues and docs and didn't see anything like this.

Is there a way to round to the nearest X of an interval with `date-fns`? For example, is there anything like this?

``` js
roundToNearestMinute(new Date())
roundToNearestMinutes(new Date(), 5)
roundToNearest(new Date(), 5, "minutes")

startOfMinutes(new Date(), 5)
startOf(new Date(), 5, "minutes")

endOfMinutes(new Date(), 5)
endOf(new Date(), 5, "minutes")
```

**Comment by vgebrev** [comment]
This works for me to round minutes:

    function roundToNearestMinutes(date, interval) {
        const roundedMinutes = Math.floor(getMinutes(date) / interval) * interval;
        return setMinutes(startOfMinute(date), roundedMinutes);
    }

    console.log(roundToNearestMinutes(new Date(), 15));

**Comment by kossnocorp** [comment]
I like the idea. 👍 for `roundToNearestMinutes`

**Comment by xkizer** [comment]
#928

**Comment by kossnocorp** [comment]
This functionality was released as `v2.0.0-alpha.23`. See the change log: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha23
